requirements/default.txt

```diff
@@ -1,7 +1,7 @@
 numpy>=1.17.3
 scipy>=1.4.1
 networkx>=2.2
-pillow>=6.1.0,!=7.1.0,!=7.1.1,!=8.3.0
+pillow>=6.1.0,!=7.1.0,!=7.1.1,!=8.3.0,<9.1.0
```

Comment:
```suggestion
pillow>=6.1.0,!=7.1.0,!=7.1.1,!=8.3.0,!=9.1.0
```

Reply:
Can we keep these epehemeral and work with upstream to fix this? This seems like a misunderstanding somewhere.